plugins/default/lib/mongodb_health.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import traceback
+from collections import Counter
+
+import openshift
+import nagios
+from mongodb import ReplsetStatuses
+
+
+def generate_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--selector', type=str, action="store",
+                        required=True, help="Label Selector")
+    return parser
+
+
+check_mongodb_cmd = ("bash", "-c",
+                     "mongo 127.0.0.1/admin "
+                     "-u \"admin\" -p \"$MONGODB_ADMIN_PASSWORD\" "
+                     "--eval=\"rs.status().myState\" --quiet")
+
+
+def parse_mongo_result(output):
+    try:
+        return int(output.rstrip("\n"))
+    except:
+        return None
```

Comment:
The output is output from the exec command, its not guaranteed that we have a mongodb result until the parsing has processed the output.  I can't use the output directly as-is because the output includes trailing new line and it shouldn't be the exec that removes that as the new-line could very well mean something in a later check.

I do think we could improve on what openshift.exec_in_pods provides us, and perhaps we should create a JIRA for this that we can bring in at the end of a sprint when we have some extra capacity.  It shouldn't hold up completely this check.  The more checks we create, the more we'll be able to decide what enhancements we can make the shared openshift module functions.